### DISCUSSION TITLE
How do I handle onFocus and onBlur event for multiple input elements using react-hook-form in a component

### DISCUSSION THREAD

**Discussion by dindustack** [react-hook-form/react-hook-form#9197] [General]
This is the child element 
import { Input, Text, View, VStack } from 'native-base'
import { Control, Controller } from 'react-hook-form'

import React from 'react'

export interface PropertyInputProps {
  defaultValue: string
  name: string
  control: Control<any>
  isFocused?: () => void
  onBlur?: () => void
}

export function PropertyInput({
  control,
  name,
  defaultValue,
  isFocused,
  onBlur
}: PropertyInputProps) {
  return (
    <View>
      <Controller
        control={control}
        name={name}
        defaultValue={defaultValue}
        render={({ field: { value, ref, onChange } }) => (
          <Input
            value={value}
            ref={ref}
            onChangeText={onChange}
            placeholder="Type..."
            onFocus={isFocused}
            onBlur={onBlur}
            type="text"
            color="type"
            background="transparent"
            editable={true}
            pl={0}
            autoComplete="off"
            variant="unstyled"
          />
        )}
      />
    </View>
  )
}

**Comment by bluebill1049** [comment]
not clear with your question and code snipt is hard to read.
